src/templates/_macros/app_tile.html

```diff
@@ -14,7 +14,7 @@
       price_only -- for Feed if only want to show price, no install
   #}
   {% set tag = 'div' if is_detail else 'a' %}
-  <{{ tag }} class="product mkt-tile{{ ' feed-app' if feed_app }}"
+  <{{ tag }} class="product mkt-tile{{ ' feed-app c' if feed_app }}"
```

Comment:
It's weird but featured apps need it since the tile is not aware of its height and doesn't fill in the white background without it. Was surprised this happens with flexed elements but there might be a floated child somewhere.